tree/tree/src/TTreeCache.cxx

```diff
@@ -57,6 +57,17 @@ The learning period is stopped (and prefetching is actually started) when:
      fEntryMin + fgLearnEntries (default to 100).
    - A 'cached' TChain switches over to a new file.
 
+Further, the TreeCache can optimize its behavior on a cache miss.  When
+miss optimization is enabled, it will track all branches utilized after
+the learning phase (those that cause a cache miss).  When one cache miss
+occurs, then all the utilized branches will be prefetched for that event.
+This optimization utilizes the observation that infrequently accessed
+branches are often accessed together.  For example, this will greatly speed
+up an analysis where the results of a trigger are read out for every branch,
+but the majority of event collections are read only when the trigger results
+pass a set of filters.  NOTE - when this mode is enabled, the memory dedicated
+to the cache will up to double in the case of cache miss.
+
```

Comment:
The PR blur states "Unfortunately, we must iterate through a set of branches and find the correct basket. This is not necessarily a cheap CPU operation and may be too expensive if the underlying filesystem is SSD-based. Hence, we turn this optimization off by default."   

Some equivalent wording ought to be in this description.